stable/external-dns/templates/deployment.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "external-dns.fullname" . }}
+  labels: {{ include "external-dns.labels" . | indent 4 }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+spec:
+  strategy:
+    type: Recreate
```

Comment:
The reason we run it this way is because ExternalDNS shouldn't run concurrently. When usign the rolling updating this can happen though, whereas `Recreate` makes sure the current pod is fully terminated before the next one spawns (at least this worked fine on Kube 1.7).

However, the impact of a temporary concurrent run shouldn't have any harming effect, so 👍 .